src/overlays/actors/ovl_Bg_Kin2_Picture/z_bg_kin2_picture.h

```diff
@@ -3,17 +3,29 @@
 
 #include "global.h"
 
+#define BG_KIN2_PICTURE_SKULLTULA_COLLECTED(thisx) (((thisx)->params >> 5) & 1)
+#define BG_KIN2_PICTURE_GETS_3FC(thisx) ((u8)(((thisx & 0x3FC)) >> 2))
+#define BG_KIN2_PICTURE_SKULLTULA_SPAWN_PARAM(thisx) ((((thisx)->params & 0x1F) << 2) | 0xFF03)
+
 struct BgKin2Picture;
 
 typedef void (*BgKin2PictureActionFunc)(struct BgKin2Picture*, PlayState*);
 
 typedef struct BgKin2Picture {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xF0];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ColliderTris colliderTris;
+    /* 0x17C */ ColliderTrisElement colliderElement;
+    /* 0x1D8 */ UNK_TYPE1 pad1D8[0x5C];
     /* 0x234 */ BgKin2PictureActionFunc actionFunc;
-    /* 0x238 */ char unk_238[0xC];
+    /* 0x238 */ s16 step;
+    /* 0x23A */ s8 paintingTimer; // used both for when painting is shaking and when it starts falling to time Gold
+                                  // Skulltula spawn.
```

Comment:
Struct comments go on one line, this one can probably be shortened a bit anyway.